apps/web/src/abis/BaseRegistrarAbi.ts

```diff
@@ -0,0 +1,453 @@
+export default [
```

Comment:
double checking that we need this and it's not a subset of an abi we've already included

Reply:
yeah only this contract has `safeTransferFrom`